src/backend/protocols/go.twitter/worker.go

```diff
@@ -12,33 +13,49 @@ import (
 	"github.com/gocql/gocql"
 	"github.com/nats-io/go-nats"
 	"sync"
+	"time"
 )
 
 type (
 	Worker struct {
-		AccountWorkers map[string]*AccountWorker // one worker per active Twitter account
-		Index          backends.LDAIndex
-		NatsConn       *nats.Conn
-		NatsSubs       []*nats.Subscription
-		Notifier       *Notifications.Notifier
-		Store          backends.LDAStore
-		WorkersGuard   *sync.RWMutex
-		Conf           WorkerConfig
+		AccountHandlers map[string]*AccountHandler // one worker per active Twitter account
+		HaltGroup       *sync.WaitGroup
+		Index           backends.LDAIndex
+		Id              string
+		NatsConn        *nats.Conn
+		NatsSubs        []*nats.Subscription
+		Notifier        *Notifications.Notifier
+		Store           backends.LDAStore
+		WorkersGuard    *sync.RWMutex
+		Conf            WorkerConfig
 	}
 
 	WorkerConfig struct {
+		Workers          uint8               `mapstructure:"workers"`
 		TwitterAppKey    string              `mapstructure:"twitter_app_key"`
 		TwitterAppSecret string              `mapstructure:"twitter_app_secret"`
 		BrokerConfig     broker.BrokerConfig `mapstructure:"BrokerConfig"`
 	}
 )
 
-func InitAndStartWorker(conf WorkerConfig) (worker *Worker, err error) {
+const (
+	failuresThreshold = 72 // how many hours to wait before disabling a faulty remote.
+	noPendingJobErr   = "no pending job"
+	pollThrottling    = 30 * time.Second
+	needJobOrderStr   = `{"worker":"%s","order":{"order":"need_job"}}`
+)
+
+func InitWorker(conf WorkerConfig, verboseLog bool, id string) (worker *Worker, err error) {
+
+	if verboseLog {
```

Comment:
maybe a configuration flag insteaf of an explicit variable (code have to be changed for activation)